src/main/kotlin/com/autonomousapps/internal/kotlin/KotlinPlatformType.kt

```diff
@@ -0,0 +1,20 @@
+package com.autonomousapps.internal.kotlin
+
+import org.gradle.api.Named
+import org.gradle.api.attributes.*
+import java.io.Serializable
+
+/** Shaded copy of the real KGP [org.jetbrains.kotlin.gradle.plugin.KotlinPlatformType] for classpath reasons. */
+enum class KotlinPlatformType: Named, Serializable {
```

Comment:
should this be public or `internal`?

Reply:
[5f179ec](https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/pull/919/commits/5f179ecc69505a94aef187b704ad44ca3ec872e6)